scripts/qesap/test/unit/test_qesap_ansible.py

```diff
@@ -50,8 +50,9 @@ def test_ansible_verbose(run, _, base_args, tmpdir, create_inventory, create_pla
     """
     provider = 'grilloparlante'
     playbooks = {'create': ['get_cherry_wood']}
+    verbosity = "true"
 
-    config_content = ansible_config(provider, playbooks)
+    config_content = ansible_config(provider, playbooks, verbosity)
```

Comment:
This test is ok, but I propose you to add one more test, not using *ansible_config* and composing, directly in the test function, a config.yaml with no *verbosity:* at all (and not only *verbosity:false* )

Reply:
I mean, what is the expected qesap.py behavior if a user omit *verbosity:* at all in its config.yaml? Maybe create one more test about this scenario.

Reply:
I just realize that you adapt an existing test but my suggestion is to create two more tests focused on the presence or absence of -vvvv in the generated ansible command, one test about starting with a config.yaml with false, another one without and verbosity key at all in the config.yaml.  You may also test that a script executed with *qesap.py --verbose* and *verbosity: false* in the config.yaml, result in Ansible called without -vvvv